bonsai/sdk/src/alpha.rs

```diff
@@ -129,8 +129,8 @@ pub mod responses {
         pub state: Option<String>,
         /// Elapsed Time
         ///
-        /// Elapsed time for a given session, in seconds
-        pub elapsed_time: Option<u64>,
+        /// Elapsed time for a given session, in seconds (f64 floating point, string encoded)
+        pub elapsed_time: Option<String>,
```

Comment:
I would think using a stronger data type than a `String` is usually preferred, seems less error prone and we can rely more on the type system. Is this just some serde-json compat issue?

Reply:
Will migrate to f64 on the bonsai side, fixed in: f594fecf612cce1407baec31ce638469ed62587f